mypy.ini

```diff
@@ -0,0 +1,13 @@
+[mypy]
+mypy_path = built
```

Comment:
I don't like having a configurable build directory name hardcoded in the mypy config file